task3/styles.css

```
@@ -0,0 +1,217 @@
+body {
+    width: 100%;
+    height: 100%;
+    padding: 0;
+    margin: 0;
+    color: #000000;
+    display: block;
+}
+body, html {
+    height: 100%;
+}
+header{
+    width: 100%;
+    position: fixed;
+    background-color: white;
+    left: 0;
+    top: 0;
+    box-shadow: rgb(220, 220, 220) 0 2px 10px;
+}
+footer{
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    flex: 0 0 auto;
+    background-color: white;
+    text-align: center;
+    color: gray;
+    border-top: 1px gray solid;
+}
+
+.header-content {
+    width: 92%;
+    max-width: 1300px;
+    margin: auto;
+    padding: 22px 0;
+    height: 36px;
+}
+.header-content::after {
+    clear: both;
+    width: 0;
+    height: 0;
+    content: '';
+    overflow: hidden;
+    display: block;
+}
+.inline-block, .right-block {
+    display: inline-block;
+    vertical-align: middle;
+}
+.right-block{
+    float: right;
+}
+
+.header_icons{

```

Именование классов должно быть однообразным
```suggestion
.header-icons{

```